rollup.config.ts

```diff
@@ -76,30 +76,29 @@ const configUMDMinified = {
   context
 }
 
-// ES module (for bundlers)
-const configEs = {
+// CommonJS (for Node) and ES module (for bundlers) build.
+// (We could have three entries in the configuration array
+// instead of two, but it's quicker to generate multiple
+// builds from a single configuration where possible, using
+// an array for the `output` option, where we can specify
+// `file` and `format` for each target)
+const configCjsAndEs = {
   input,
-  output: {
-    file: pkg.module,
-    format: 'es',
-    sourcemap: true
-  },
-  plugins: browserPlugins,
-  external,
-  context
-}
-
-// CommonJS (for Node)
-const configCjs = {
-  input,
-  output: {
-    file: pkg.main,
-    format: 'cjs',
-    sourcemap: true
-  },
+  output: [
+    {
+      file: pkg.main,
+      format: 'cjs',
+      sourcemap: true
+    },
+    {
+      file: pkg.module,
+      format: 'es',
```

Comment:
When packaging up `webnative-filecoin`, it uses the `module` entry not the `main` entry. So I'm building both of these for node.

I'm still hopelessly confused by js module formats lol. would it be wrong for us to just ditch either CJS or ES entirely? UMD for browser build & either cjs/es for node? Would also speed up build times.

Related node: I don't think anything uses or minified build right now. We could either remove it from the build or point `browser` (in `pacakge.json` to the minified build instead of the full umd

Reply:
I think `browser` in the package.json should point to the minified build :+1: 

> I'm still hopelessly confused by js module formats lol. would it be wrong for us to just ditch either CJS or ES entirely? UMD for browser build & either cjs/es for node? Would also speed up build times.

We should have both cjs and mjs. The problem here is that there are users that use webnative with ES modules and users that use webnative with commonjs modules.
I think we should reconsider that once node supports esnext natively and by default & the old node versions that don't support it are out of LTS. That'll take a while, if it happens at all, though :D